### ISSUE TITLE
Installation outdated

### ISSUE THREAD

**Issue by radholm** [hplusai/openassistant#3]
Can't install this by following current installation guide

**Comment by vavrek** [comment]
Yes, sorry. This repository is seriously outdated. The most recent is on GitLab: https://gitlab.com/open-assistant/oa-arch

Even the installation guide on GitLab is a headache and there are certainly better ways. Am just learning about 'package.json' files and such. Could use help in this regard, how to easily install dependencies, or even better, create some packages. Working towards this!
